tests/unit/test_utils.py

```diff
@@ -0,0 +1,22 @@
+"""
+Copyright (c) 2022 Aiven Ltd
+See LICENSE for details
+"""
+from pathlib import Path
+from tests.utils import get_clickhouse_version
+from typing import List, Union
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.mark.parametrize(
+    "command,output,expected",
+    [
+        ([Path("/usr/bin/clickhouse"), "server"], b"ClickHouse server version 22.0.0.1 (official build).", (22, 0, 0, 1)),
+        ([Path("/usr/local/bin/clickhouse", "server")], b"ClickHouse server version 22.1.2.3.", (22, 1, 2, 3)),
+    ],
+)
+def test_get_clickhouse_version(command: List[Union[str, Path]], output: bytes, expected: tuple[int]):
```

Comment:
Here too, the type of of `expected` is not `tuple[int]` (and the return type is missing).

Reply:
Ack